.vscode/extensions.json

```diff
@@ -4,6 +4,7 @@
     "esbenp.prettier-vscode",
```

Comment:
There is a Firefox debugger extension for VS Code, but I have omitted for the time being because I was unable to get it running on my Mac. I will probably take a stab at this again when I have a little more time.